docs/sources/mimir/visualize/query-metric-labels.md

```diff
@@ -0,0 +1,95 @@
+---
+description: Learn how to query metric labels from Grafana Mimir.
+title: Query metric labels
+menuTitle: Query metric labels
+weight: 2
+keywords:
+  - labels
+---
+
+# Query metric labels
+
+Grafana Mimir supports multiple [HTTP API]({{< relref "../references/http-api/index.md" >}}) endpoints to query metric label names and values.
+In this document we explain the main differences and trade-off between these API endpoints.
+
+## Querying label names
+
+The following API endpoints can be used to find the list of label names of all (or a subset of) series stored in Mimir:
+
+- **[Get label names]({{< relref "../references/http-api/index.md#get-label-names" >}})**<br />
+  `GET,POST <prometheus-http-prefix>/api/v1/labels`
+- **[Label names cardinality]({{< relref "../references/http-api/index.md#label-names-cardinality" >}})**<br />
+  `GET,POST <prometheus-http-prefix>/api/v1/cardinality/label_names`
+
+> **Note**: `<prometheus-http-prefix>` is the Prometheus HTTP prefix as documented in the [HTTP API]({{< relref "../references/http-api/index.md#path-prefixes" >}}) reference.
+
+### Features comparison
+
+The different API endpoints have different features.
+The following list summarises the main differences.
+
+#### Can specify the time range in the request?
+
+- **Get label names**<br />
+  Yes. However, the actual time range granularity is restricted to TSDB block range periods (2h for the most recent metrics and up to 24h for historical metrics when running Mimir with the default configuration). In practice, the input `start` and `end` parameters are rounded to the TSDB block boundaries: the `start` parameter is rounded to the start of the nearest block containing samples whose timestamp is lower or equal than the input timestamp, while the `end` parameter is rounded to the end of the nearest block containing samples with timestamp greater or equal than the input timestamp.
```

Comment:
Can these be done without the html `<br />` or did you do them intentionally? I think replacing `<br />` with a `new-line` shouldn't change the rendering.

Reply:
I can replace `<br />` with an empty line, but when compiled the spacing is a bit different and looks nicer with `<br />`.